samples/subsys/video/capture/src/main.c

```diff
@@ -104,6 +138,16 @@ int main(void)
 		printk("\rGot frame %u! size: %u; timestamp %u ms\n", frame++, vbuf->bytesused,
 		       vbuf->timestamp);
 
+#if DT_HAS_CHOSEN(zephyr_display)
```

Comment:
Could you please make the content a function (e.g `video_display_frame()`) ? (and no-op if !zephyr_display)

Reply:
yes, done. I also made a display_setup() function. Thanks